packages/twenty-server/src/engine/metadata-modules/remote-server/remote-table/remote-table.service.ts

```diff
@@ -574,20 +447,154 @@ export class RemoteTableService {
     }
   }
 
-  private buildReferencedTable(
-    remoteServer: RemoteServerEntity<RemoteServerType>,
-    remoteTableInput: RemoteTableInput,
-  ): ReferencedTable {
-    switch (remoteServer.foreignDataWrapperType) {
-      case RemoteServerType.POSTGRES_FDW:
-        return {
-          table_name: remoteTableInput.name,
-          schema_name: remoteServer.schema,
-        };
-      case RemoteServerType.STRIPE_FDW:
-        return { object: remoteTableInput.name };
-      default:
-        throw new BadRequestException('Foreign data wrapper not supported');
+  private async getDistantTablesWithUpdates({
```

Comment:
This logic looks huge. Do you think it would make sense to have a `RemoteTableSchemaUpdatesService`?